docs/configuration-reference/components/metallb.md

```diff
@@ -50,8 +50,8 @@ MetalLB component configuration example:
 ```tf
 component "metallb" {
   address_pools = {
-    default = ["147.63.8.20/32"]
-    special_addresses = ["147.85.47.16/29", "147.85.47.24/29"]
+    pool1 = ["147.63.8.20/32"]
```

Comment:
How can you choose which pool will be used? Maybe we could put some link in the description?

Reply:
This is covered by the official MetalLB docs. TL;DR: you specify an annotation on the relevant service. I've included a "for more info" note with a link to the official docs.

Reply:
Hm, as a person who is not very familiar with MetalLB, having some more specific links and description of what this flag does would be useful.

Could we link to https://metallb.universe.tf/usage/example/? Or maybe give an example of setting this annotation to some component service, if that's currently supported?

Reply:
+1 on @invidian's comment about linking an example.

But truth be told, couple of lines of addition in our documentation would make much more sense that merely linking the example from metallb docs.


Example: Annotate the Service with the name of the address pool to obtain an IP address within the specified range:
```
metallb.universe.tf/address-pool: pool1
```

Reply:
The question is - if we start including examples for MetalLB **usage**, why document just this case? There are many more params which can be tweaked and we would be duplicating the MetalLB docs.

Reply:
We should document this case, because `metallb.universe.tf/address-pool` Service labels is directly related to `address_pools` feature and key field in it IMO.

Reply:
But there is more info regarding address pools. The MetalLB docs explain how allocation works, how to disable auto-allocation etc. Any info we include there would be duplicating the MetalLB docs.

Reply:
I'm fine with some more specific link to MetalLB documentation too. We don't necessarily need to duplicate the content here. But linking to https://metallb.universe.tf/ is not very helpful, as the documentation is broad.

Reply:
We can do a TL;DR version in our documentation and link to metallb docs for more comprehensive documentation.

Reply:
My point is - if we do what you're proposing @invidian @ipochi, why do it just for pool selection? What's special about this feature? There is a bunch of MetalLB functionality which isn't covered in the Lokomotive docs.

Reply:
All other knobs we expose for MetalLB are generic to any Kubernetes workload (selector and tolerations). Only  `address_pools` is MetalLB specific, hence the request to have a better documentation for it.

Reply:
Fine, I'll add some info.

Reply:
Done.